prime-router/settings/staging/0035-ma-secondary.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Added a secondary feed so one of our senders could send OTC data to MA in staging, as a test.